ddtrace/contrib/flask/patch.py

```diff
@@ -279,6 +282,42 @@ def traced_start_response(status_code, headers):
     return traced_start_response
 
 
+@with_instance_pin
+def traced_stream_with_context(pin, wrapped, instance, args, kwargs):
```

Comment:
Avoid using stream_with_context. Provide support for different wsgi streamed response.